frontend/components/forms/fields/Dropdown/_styles.scss

```diff
@@ -93,7 +93,7 @@
     .Select-value-label {
       font-size: $small;
       color: $core-fleet-black;
-      line-height: 28px;
```

Comment:
This isn't good because line-height changes for various dropdowns (e.g. Team dropdown)